toolkit/scripts/analysis.mk

```
@@ -71,6 +75,17 @@ sodiff-repo: $(SODIFF_REPO_FILE)
 
 $(SODIFF_REPO_FILE):
 	echo $(SODIFF_REPO_SOURCES) | sed -E 's:([^ ]+[.]repo):$(SPECS_DIR)/mariner-repos/\1:g' | xargs cat > $(SODIFF_REPO_FILE)
+	awk '{if ($$0 ~ /enabled=0/) $$0="enabled=1"; print}' $(SODIFF_REPO_FILE) > $(SODIFF_REPO_FILE)_fixed
+	mv $(SODIFF_REPO_FILE)_fixed $(SODIFF_REPO_FILE)
+
+# sodiff-repo-extended: Generate just the sodiff.repo file
+.PHONY: sodiff-repo-extended
+sodiff-repo-extended: $(SODIFF_REPO_FILE_EXTENDED)
+
+$(SODIFF_REPO_FILE_EXTENDED):
+	echo $(SODIFF_REPO_SOURCES_EXTENDED) | sed -E 's:([^ ]+[.]repo):$(SPECS_DIR)/mariner-repos/\1:g' | xargs cat > $(SODIFF_REPO_FILE_EXTENDED)
+	awk '{if ($$0 ~ /enabled=0/) $$0="enabled=1"; print}' $(SODIFF_REPO_FILE_EXTENDED) > $(SODIFF_REPO_FILE_EXTENDED)_fixed
+	mv $(SODIFF_REPO_FILE_EXTENDED)_fixed $(SODIFF_REPO_FILE_EXTENDED)
```

Similar idea for simplification here:
```suggestion
	sed -iE 's/enabled=0/enabled=1/g' $(SODIFF_REPO_FILE_EXTENDED)
```